packages/components/src/components/Trigger/Trigger.js

```diff
@@ -0,0 +1,167 @@
+import { Link } from 'react-router-dom';
+import './Trigger.scss';
+import { Table } from '@tektoncd/dashboard-components';
+
+const Trigger = ({ intl, eventListenerNamespace, trigger }) => {
+  /* istanbul ignore next */
+  function toTriggerTemplate() {
+    window.location.href = `../#/namespaces/${eventListenerNamespace}/triggertemplates/${
+      trigger.template.name
+    }`;
+  }
+
+  /* istanbul ignore next */
+  function toTriggerBinding() {
+    window.location.href = `../#/namespaces/${eventListenerNamespace}/triggerbindings/${
+      trigger.binding.name
+    }`;
+  }
+
+  const tableHeaders = [
+    {
+      key: 'name',
+      header: intl.formatMessage({
+        id: 'dashboard.tableHeader.name',
+        defaultMessage: 'Name'
+      })
+    },
+    {
+      key: 'value',
+      header: intl.formatMessage({
+        id: 'dashboard.tableHeader.value',
+        defaultMessage: 'Value'
+      })
+    }
+  ];
+
+  let triggerParams = [];
+
+  if (trigger.params) {
+    triggerParams = trigger.params.map(param => ({
+      id: param.name,
+      name: param.name,
+      value: param.value
+    }));
+  }
+
+  let interceptorValues = [];
+
+  if (trigger.interceptor.header) {
+    interceptorValues = trigger.interceptor.header.map(header => ({
+      id: header.name,
+      name: header.name,
+      value: header.value
+    }));
+  }
+
+  return (
+    <>
+      <h5>
+        {intl.formatMessage({
+          id: 'dashboard.triggerDetails.triggerBinding',
+          defaultMessage: 'TriggerBinding'
+        })}
+      </h5>
+      <Button
+        kind="ghost"
+        renderIcon={LinkIcon}
+        className="link-btn"
+        size="small"
+        onClick={toTriggerBinding}
+      />
+      {trigger.binding.name}
+      <br /> <br />
+      <h5>
+        {intl.formatMessage({
+          id: 'dashboard.triggerDetails.triggerTemplate',
+          defaultMessage: 'TriggerTemplate'
+        })}
+      </h5>
+      <Button
+        kind="ghost"
+        renderIcon={LinkIcon}
+        className="link-btn"
+        size="small"
+        onClick={toTriggerTemplate}
+      />
+      <Link
+        to={urls.triggerTemplates.byName({
+          namespace: eventListenerNamespace,
+          triggerTemplateName: trigger.template.name
+        })}
+      >
+        {trigger.template.name}
+      </Link>
+      <br /> <br />
+      <h5>
+        {intl.formatMessage({
+          id: 'dashboard.triggerDetails.triggerParameters',
+          defaultMessage: 'Parameters'
+        })}
+      </h5>
+      <Table
+        headers={tableHeaders}
+        rows={triggerParams}
+        emptyTextAllNamespaces={intl.formatMessage({
+          id: 'dashboard.trigger.noParams',
+          defaultMessage: 'No parameters found for this Trigger.'
+        })}
+        emptyTextSelectedNamespace={intl.formatMessage({
+          id: 'dashboard.trigger.noParams',
+          defaultMessage: 'No parameters found for this Trigger.'
+        })}
+        isSortable
```

Comment:
Carbon doesn't seem to like nested Tables, especially when sorting is enabled. Removing `isSortable` resolves the weird table header layout. The hover styles still need work but at least the headers are consistent.